frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.mdx

```diff
@@ -0,0 +1,104 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import { Heading, Paragraph } from '@digdir/design-system-react';
+import * as StudioTableLocalPaginationStories from './StudioTableLocalPagination.stories';
+
+<Meta of={StudioTableLocalPaginationStories} />
+
+<Heading level={1} size='small'>
+  StudioTableLocalPagination
+</Heading>
+<Paragraph>StudioTableLocalPagination is a wrapper for StudioTableRemotePagination.</Paragraph>
+<Paragraph>
+  <ul>
+    <li>
+      Use this component when you are fetching a complete set of data, and want the component to
+      handle pagination and sorting internally. Otherwise, use `StudioTableRemotePagination`.
+    </li>
+    <li>
+      Pagination and sorting can be removed by not passing props to `pagination` and setting
+      `isSortable` to `false`.
+    </li>
+  </ul>
+</Paragraph>
```

Comment:
We should not wrap the entire `<ul>` in a paragraph.
```suggestion
  <ul>
    <li>
      Use this component when you are fetching a complete set of data, and want the component to
      handle pagination and sorting internally. Otherwise, use `StudioTableRemotePagination`.
    </li>
    <li>
      Pagination and sorting can be removed by not passing props to `pagination` and setting
      `isSortable` to `false`.
    </li>
  </ul>
```

Reply:
Fixed 😊